content/examples.md

```diff
@@ -15,6 +15,10 @@
 
 ---
 
+(interactive-data-fitting)=
```

Comment:
what's this?

Reply:
A sphinx link target so that we can link to a section and the link will survive file moves and renames: https://myst-parser.readthedocs.io/en/latest/using/syntax.html#targets-and-cross-referencing